aspects/electrical/PowerBus/test/UtPowerBusElect.cpp

```diff
@@ -417,7 +417,7 @@ void UtPowerBusElect::testOneLoad()
 void UtPowerBusElect::testLargeLoadNumber()
 {
     //set numLoads = 10,000
-    tConfigData->mNumLoads=10,000;
+    tConfigData->mNumLoads=10000;
```

Comment:
This fix is causing the ASAN error that's breaking the unit test (ASan) CI job.  Maybe due to using too much memory with a very large number of loads... or something:

```
==971763==ERROR: AddressSanitizer: heap-buffer-overflow on address 0x61d00000ee88 at pc 0x0000004cdad6 bp 0x7ffdedd75120 sp 0x7ffdedd75110
```

The CI job broke instead of printing out the error, but i get that when i make your fix and run the test it on my own.  No errors if the value is just set to 10 which I think is what that old code was actually doing, but the point of the test is to test really large number of loads.  So for now I would like you to just change this to 10 instead of 10000 and resubmit, and I'll take an action to fix this later.   Or maybe i'll just deprecate PowerBusElect and delete this whole test.  PowerBusElect is an obsolete link and can't be used from GunnsDraw anyway.